packages/generator/src/generator.js

```diff
@@ -26,7 +26,7 @@ export default class Generator {
     this.distPath = this.options.generate.dir
     this.distNuxtPath = path.join(
       this.distPath,
-      isUrl(this.options.build.publicPath) ? '' : this.options.build.publicPath
+      isUrl(this.options.build.publicPath) ? this.options.build.publicPathFolder : this.options.build.publicPath
```

Comment:
What if instead of introducing a new option, using `pathname` of `publicPath`? (so if `publicPath` is `https://cdn.com/something/_nuxt`, we set it to `dist/something/_nuxt`)

Reply:
Sound better and cleaner.